CMakeLists.txt

```diff
@@ -46,6 +46,12 @@ if(NOT DEFINED INSTALL_RUNKRATOS)
   set(INSTALL_RUNKRATOS ON)
 endif(NOT DEFINED INSTALL_RUNKRATOS)
 
+if(NOT DEFINED REMOVE_INSTALL_DIRECTORIES)
+  message("-- No REMOVE_INSTALL_DIRECTORIES is defined, setting to ON")
+  set(REMOVE_INSTALL_DIRECTORIES ON)
+endif(NOT DEFINED REMOVE_INSTALL_DIRECTORIES)
```

Comment:
option(REMOVE_INSTALL_DIRECTORIES "Remove the install directories to achieve a clean compilation" ON)